.diffcover.toml

```diff
@@ -0,0 +1,4 @@
+[tool.diff_cover]
+exclude = [
+    "*/chia/_tests/*/config.py",
```

Comment:
I would have expected it to take `**` rather than `*` to cover all sub directories

Reply:
So yeah, it's a bit curious and I ended up reading the source to figure it out.  It uses glob for includes but fnmatch for excludes.  Glob gives you `**` but fnmatch doesn't.  Additionally, for excludes it first compares the pattern to the basename and then to the absolute path...  Interesting situation that led to this somewhat odd pattern.  Eventually.